launchflow/aws/lambda_service.py

```diff
@@ -1,88 +1,264 @@
+import inspect
+import os
+import shutil
+import subprocess
+import tempfile
 from dataclasses import dataclass
-from typing import List, Optional
+from typing import IO, Any, Dict, List, Optional, Union
 
+import requests
 from typing_extensions import Callable
 
 import launchflow as lf
 from launchflow import exceptions
-from launchflow.aws.acm import ACMCertificate
-from launchflow.aws.lambda_function import LambdaFunction
-from launchflow.aws.service import (
-    AWSDockerService,
-    AWSDockerServiceOutputs,
-    AWSStaticService,
+from launchflow.aws.api_gateway import (
+    CORS,
+    APIGateway,
+    APIGatewayLambdaIntegration,
+    APIGatewayRoute,
 )
+from launchflow.aws.lambda_function import (
+    LambdaFunction,
+    LambdaFunctionURL,
+    LambdaRuntime,
+)
+from launchflow.aws.service import AWSService
+from launchflow.config import config
 from launchflow.models.enums import ServiceProduct
+from launchflow.models.flow_state import AWSEnvironmentConfig
+from launchflow.models.launchflow_uri import LaunchFlowURI
 from launchflow.node import Inputs
 from launchflow.resource import Resource
-from launchflow.service import StaticServiceOutputs
+from launchflow.service import ServiceOutputs
+from launchflow.workflows.utils import zip_source
+
+
+def _get_relative_handler_import_path(func: Callable) -> str:
+    # Get the absolute path of the directory containing the launchflow.yaml file
```

Comment:
These per line comments seem a bit excessive, maybe just comment at the top of the function what it's doing?

Reply:
this is a chatgpt artifact, so I honestly kinda prefer leaving them so we can have the original intention of the code in case we need to debug this later